spacy/language.py

```diff
@@ -1876,10 +1872,36 @@ def from_config(
         if vocab_b is not None:
             nlp.vocab.from_bytes(vocab_b)
 
+        def fetch_pipes_status(
+            value: Union[str, Iterable[str]], key: str
+        ) -> Iterable[str]:
+            """Fetch value for `enable` or `disable` w.r.t. the specified config and passed arguments passed to
+            .load(). If both arguments and config specified values for this field, the passed arguments take precedence
+            and a warning is printed.
+            value (Optional[Union[str, Iterable[str]]]): Passed value for `enable` or `disable`.
+            key (str): Key for field in config (either "enabled" or "disabled").
+            RETURN (Iterable[str]):
+            """
+            # We assume that no argument was passed if the value is the default value of an empty SimpleFrozenList.
+            if isinstance(value, SimpleFrozenList) and len(value) == 0:
```

Comment:
I'm not really happy about this way of inferring that no value was passed for this argument. An alternative would be setting the default values for `enable` and `disable` to `None`, but this would have to be done for multiple functions.

Reply:
Hm, it's not great, especially because of the indirect dependency across almost 200 lines of code.
What if we store one of these frozen lists in a private global var, specify that as default in the `from_config` signature and check for identity here instead of `isinstance`? Then at least this check becomes more meaningful and correct, and because of the way `SimpleFrozenList` is implemented I don't think there could be any harm in reusing the var for all three input vars here? We could also have 3 global vars - one for `disable`/`enable`/`exclude` each.